crates/networking/src/rpc_abi.rs

```diff
@@ -99,12 +99,13 @@ pub struct BlockWithHash {
     pub transactions: Vec<TransactionWithHash>,
 }
 
-#[derive(Debug, Deserialize, Serialize)]
+#[derive(Debug, Deserialize, Serialize, Clone)]
 pub struct RpcSetAccountHeadRequest {
     pub account: String,
     pub start: String,
     pub end: String,
     pub blocks: Vec<BlockWithHash>,
```

Comment:
This is unideal since this RPC type no longer matches the ironfish request type, but there is no wrapper type around the encrypted message sent back to the server, but I didn't want to make that additional change here.